### DISCUSSION TITLE
Expand searching Teams beyond the first 1000 results

### DISCUSSION THREAD

**Discussion by yuandongjian** [grafana/grafana#39670] [General]
I have 1296 teams, but I can't search the last 296 teams because the frontend is stuck looking only for the first 1000 teams and there is no page button，and Search Team is a front-end search, it's a fake.
which is a low-level problem.

![image](https://user-images.githubusercontent.com/35158271/134463517-410c6a25-0884-4cf4-a995-ddc080310080.png)

**Comment by jackw** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by yuandongjian** [comment]
I think this a bug, not a feature. @jackw

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
